.tekton/release-pipeline.yaml

```diff
@@ -77,8 +77,8 @@ spec:
                   env TARGET_OPENSHIFT=true bash hack/generate-releaseyaml.sh > /tmp/release.yaml
                   msg="pac release ${version} on branch ${target}"
                   set +x
-                  echo python hack/upload-file-to-github.py --message "Release yaml generated for ${msg}" --owner-repository openshift-pipelines/pipelines-as-code --token "TOKEN" --to-ref=refs/heads/${target} --from-ref=refs/tags/${version} -f release.k8s.yaml:release.k8s.yaml -f release.yaml:release.yaml -f ${TARGET_BRANCH}:docs/content/VERSION
-                  python hack/upload-file-to-github.py --message "Release yaml generated for ${msg}" --owner-repository openshift-pipelines/pipelines-as-code --token ${HUB_TOKEN} --to-ref=refs/heads/${target} --from-ref=refs/tags/${version} -f /tmp/release.k8s.yaml:release.k8s.yaml -f /tmp/release.yaml:release.yaml -f ${TARGET_BRANCH}:docs/content/VERSION
+                  echo python hack/upload-file-to-github.py --message "Release yaml generated for ${msg}" --owner-repository openshift-pipelines/pipelines-as-code --token "TOKEN" --to-ref=refs/heads/${target} --from-ref=refs/tags/${version} -f release.k8s.yaml:release.k8s.yaml -f release.yaml:release.yaml -f ${TARGET_BRANCH}:docs/content/VERSION -f "${TARGET_BRANCH}:pkg/params/version/version.txt"
```

Comment:
we need to put the version also in `version.txt`

Reply:
ohh I see TARGET_BRANCH containers the version from the release branch after removing the `release-`, the name was confusing

Reply:
yeah it's badly named